phygnn/layers/custom_layers.py

```diff
@@ -1527,23 +1539,42 @@ def call(self, x, hi_res_feature=None):
         hi_res_feature : tf.Tensor | np.ndarray
             This should be a 4D array for spatial enhancement model or 5D array
             for a spatiotemporal enhancement model (obs, spatial_1, spatial_2,
-            (temporal), 1). This is NaN where there are no observations and
-            real values where observations exist.
+            (temporal), features). This is NaN where there are no observations
+            and real values where observations exist.
+        exo_data : tf.Tensor | np.ndarray
+            This is an array of exogenous data used to imform the embedding,
+            like topography
```

Comment:
Maybe specify why you'd do `hi_res_feature=None` and `exo_data=None` in the docstrings since this is default?